testing/test_base.sh

```diff
@@ -50,35 +50,38 @@ echo switch ping $count | tee -a $TEST_RESULTS
 echo %%%%%%%%%%%%%%%%%%%%%% Mud profile tests | tee -a $TEST_RESULTS
 cp misc/system_muddy.conf local/system.conf
 
-device_traffic="tcpdump -en -r inst/run-port-01/scans/monitor.pcap port 47808"
-device_bcast="$device_traffic and ether broadcast"
-device_ucast="$device_traffic and ether dst 9a:02:57:1e:8f:02"
-device_xcast="$device_traffic and ether dst 9a:02:57:1e:8f:03"
-cntrlr_traffic="tcpdump -en -r inst/run-port-02/scans/monitor.pcap port 47808"
-cntrlr_bcast="$cntrlr_traffic and ether broadcast"
-cntrlr_ucast="$cntrlr_traffic and ether dst 9a:02:57:1e:8f:01"
-cntrlr_xcast="$cntrlr_traffic and ether dst 9a:02:57:1e:8f:03"
+port_1_traffic="tcpdump -en -r inst/run-port-01/scans/monitor.pcap port 47808"
+port_1_bcast_2="$port_1_traffic and ether src 9a:02:57:1e:8f:02 and ether broadcast"
+port_1_ucast_2="$port_1_traffic and ether src 9a:02:57:1e:8f:02 and ether dst 9a:02:57:1e:8f:01"
+port_1_bcast_3="$port_1_traffic and ether src 9a:02:57:1e:8f:03 and ether broadcast"
+port_1_ucast_3="$port_1_traffic and ether src 9a:02:57:1e:8f:03 and ether dst 9a:02:57:1e:8f:01"
```

Comment:
I'm wondering if we should be pedantic here and also check for a ucast that's NOT destined for :01 -- e.g. make sure that we don't receive unintended packets?  so a port_1_ncast_2 (and _3) that has a not ether dst filter on it... ?

Reply:
I think with a specific dst MAC filter it can filter out the broadcast traffic. Or what we can do is something like "ether src xxxx:01 and **not** ether broadcast"?

Reply:
I think specifying the destination will also help filter out the flooding packet for unlearned destination.